compilers/concrete-compiler/compiler/README.md

```diff
@@ -11,6 +11,16 @@ The source of the project is located in the `compiler` directory.
 cd compiler
 ```
 
+### Prerequisite: Install build tools
+
+If you intend to build the Compiler from source, you should make sure to have [CMake](https://cmake.org/) and [Ninja](https://ninja-build.org/) installed. Optionally, you can also install [Ccache](https://ccache.dev/) to speedup compilations after small changes.
```

Comment:
I think we can specify build-essentials and .or the equivalent for fedora based.
We need also python-devel, numpy, pybind11 and perl

Reply:
why perl? Python packages are listed down in the python section

Reply:
for Mac homebrew it's cmake (`brew install cmake`)

Reply:
I don't known exactly why, but is needed for the build system, for openmp project I guess. If you don't install it you have some trouble with FindBin or something like that.

```
git grep FindBin
clang/tools/scan-build/bin/scan-build:use FindBin qw($RealBin);
clang/tools/scan-build/libexec/ccc-analyzer:use FindBin;
clang/tools/scan-build/libexec/ccc-analyzer:if ($FindBin::Script =~ /c\+\+-analyzer/) {
openmp/runtime/tools/check-depends.pl:use FindBin;
openmp/runtime/tools/check-depends.pl:use lib "$FindBin::Bin/lib";
openmp/runtime/tools/check-execstack.pl:use FindBin;
openmp/runtime/tools/check-execstack.pl:use lib "$FindBin::Bin/lib";
openmp/runtime/tools/check-instruction-set.pl:use FindBin;
openmp/runtime/tools/check-instruction-set.pl:use lib "$FindBin::Bin/lib";
openmp/runtime/tools/generate-def.pl:use FindBin;
openmp/runtime/tools/generate-def.pl:use lib "$FindBin::Bin/lib";
openmp/runtime/tools/lib/LibOMP.pm:    # $FindBin::Bin is not used intentionally because it gives real path. I want to use absolute,
openmp/runtime/tools/lib/LibOMP.pm:    use FindBin;
openmp/runtime/tools/lib/LibOMP.pm:    use lib "$FindBin::Bin/lib";
openmp/runtime/tools/lib/tools.pm:    use FindBin;
openmp/runtime/tools/lib/tools.pm:    use lib "$FindBin::Bin/lib";
openmp/runtime/tools/lib/tools.pm:use FindBin;
openmp/runtime/tools/lib/tools.pm:$tool = $FindBin::Script;
openmp/runtime/tools/message-converter.pl:use FindBin;
openmp/runtime/tools/message-converter.pl:use lib "$FindBin::Bin/lib";
```